app/auth/routes.py

```diff
@@ -153,52 +153,54 @@ def register():
 
     if not r:
         raise InvalidUsageError(
-            message="Email and password must be included in the request body."
+            message="Parameters must be included in the request body."
```

Comment:
I think the previous one was a more descriptive error?

Reply:
It was, but there are only 2 parameters and we have 5 of them total that should be included. Do you think we should specify all 5?

Reply:
Actually, I think it makes the most sense to say JSON is required, as this is not parameter specific but just whether or not JSON is included and the frontend team is the only group accessing this endpoint. Individual missing params will get caught in the forloop.